hack/common.sh

```diff
@@ -125,18 +144,19 @@ cleanup_WMCO() {
   fi
   transform_csv $OPERATOR_IMAGE REPLACE_IMAGE
 
-  # Remove the operator from openshift-windows-machine-config-operator namespace
-  oc delete -f deploy/namespace.yaml
+  # Remove the openshift-windows-machine-config-operator namespace
+  oc delete ns $WMCO_DEPLOY_NAMESPACE
 }
 
 # returns the operator version in `Version+GitHash` format
 # we are just checking the status of files that affect the building of the operator binary
 # the files here are selected based on the files that we are transferring while building the
 # operator binary in `build/Dockerfile`
+# Takes a semver as an argument. This should be the version present in the WMCO CSV.
```

Comment:
So you are making the change but there is no change with the call sites at least in this commit. I briefly tried finding the commit where you are introducing the change and gave up 😃  And it is also confusing that `get_version()` now takes a version as input.

Reply:
Split the change and the callsite into a seperate commit than this to remove confusion.